.github/workflows/build-images.yml

```
@@ -285,10 +285,9 @@ jobs:
       uses: spilchen/switch-case-action@v2
       id: v2_vertica_image
       with:
-        default: ghcr.io/${{ env.OWNER_LC }}/vertica-k8s:${{ github.sha }}-v2
+        default: verticadocker/vertica-k8s-private:latest-master
```

I think this needs to prefix with docker.io. There is other code in the workflow that looks at the prefix to know it has to log into dockerhub
```suggestion
        default: docker.io/verticadocker/vertica-k8s-private:latest-master
```

---

Sure!